frontend/ui/size.lua

```diff
@@ -64,6 +64,7 @@ local Size = {
         thin = Screen:scaleBySize(0.5),
         medium = Screen:scaleBySize(1),
         thick = Screen:scaleBySize(1.5),
+        non_touch_active = Screen:scaleBySize(5),
         progress = Screen:scaleBySize(7),
```

Comment:
If we're using it always, it doesn't need to be named "non_touch_*".
We need to find another adjective for that little bar indicating an item is focused/selected.
(I would say you being English is the best to find one, but this could lead us into complications :))

Reply:
`active selection indicator`, `focus underline`, `focus bar` have your pick

Reply:
`Size.line.focus` then

Reply:
just focus? surely you could have said that from the start